modules/gathering-clusterversion-history.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history_{context}"]
+= Gathering ClusterVersion history 
+
+When updating your cluster, the cluster version operator (CVO) creates a `status.history` entry to record the update. To gather and understand the history of your successful or failed cluster update, it is important to retrieve the ClusterVersion history.
```

Comment:
nit: "failed" sounds so final.  The CVO has `Failing`, which is "we're currently having trouble", but the CVO will never give up on trying to reconcile the target release it's been pointed at.  And the `To gather...` sentence here seems a bit tautological ("to gather the history, you need to gather the history"), without explaining how gathering the history will help accomplish something else.  Maybe rephrase to something like:

> When updating your cluster, the cluster version operator (CVO) creates a `status.history` entry to record the update.  This can be useful to decide if history-related bugs apply to your cluster.  It can also be used to correlate changes in cluster behavior with one potential trigger, although correlation alone does not imply causation.